azure-kusto-data/tests/test_helpers.py

```diff
@@ -11,28 +12,51 @@
 import numpy
 
 
-def test_dataframe_from_result_table():
-    """Test conversion of KustoResultTable to pandas.DataFrame, including fixes for certain column types"""
+class PandasTests(unittest.TestCase):
```

Comment:
I just noticed you created a class for this test - you don't need to do this. Notice how the rest of the tests don't have classes.
We use pytest for tests instead of unittest, so don't import/use it